modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/APIApplicationManagementToolbar.tsx

```diff
@@ -87,7 +95,7 @@ export function APIApplicationManagementToolbar({
 				<ClayManagementToolbar.ItemList>
 					<ClayButton.Group key={1} spaced>
 						<ClayButton
-							borderless={true}
+							borderless={handleBorderless()}
```

Comment:
I think we should just delete the `borderless` prop and have the button always with a border. Why should the button look different depending on publish status?

Reply:
Hey, after talking with the team, as you probably saw, your solution is right! Cancel has no border if it is the third button :clap:

**But!** I see `handle` as a sufix added to function names when the function handles some event. In this case there is no event, and what the function does is not directly related to the border, it just checks if the API its unpublished. So a proper name to this function could be `isUnpublished`.

**But!** `itemData.applicationStatus?.key === 'unpublished'` already returns the boolean we want. We could just store it in `isUnpublished`, like this `const isUnpublished = itemData.applicationStatus?.key === 'unpublished'`.

**But!** It is a short expression, used only once, so we should just use inline `borderless={itemData.applicationStatus?.key === 'unpublished'}`

Reply:
Great!! @cgmonte Thanks for your review, make sense a lot